### ISSUE TITLE
Erreur de traduction

### ISSUE THREAD

**Issue by babastienne** [GeotrekCE/Geotrek-admin#3562]
![image](https://user-images.githubusercontent.com/16348048/231772987-6e8f1268-159f-4aa3-8e90-e5a1d766f611.png)

Dans la légende, Signage devrait être en français : Signalétique
Signalé par le Parc National de Calanques

**Comment by camillemonchicourt** [comment]
Oui, déjà remonté suite à l'implémentation de la fonctionnalité - https://github.com/GeotrekCE/Geotrek-admin/issues/1300#issuecomment-1266960394

**Comment by camillemonchicourt** [comment]
Sur les traductions, il y a aussi un manque au niveau des nouvelles fonctionnalités de duplication - https://github.com/GeotrekCE/Geotrek-admin/issues/3105#issuecomment-1404323251
